frontend/components/sidebar/left/SidebarLeftIndex.vue

```diff
@@ -55,7 +51,10 @@
             :active="button.active"
           />
         </li>
-        <li v-if="pageType === 'event'" v-for="button in eventButtons">
+        <li
+          v-if="pageType === 'event'"
```

Comment:
Note: based on the [Vue styleguide](https://vuejs.org/style-guide/) this should be split. We should have a parent `v-if` `<div>`. Was reading through it the other day :)